macos/FluentUI/Button.swift

```diff
@@ -0,0 +1,188 @@
+//
+// Copyright Microsoft Corporation
+//
+
+import AppKit
+
+fileprivate struct Constants {
+	
+	static let borderWidth: CGFloat = 1
+	
+	static let cornerRadius: CGFloat = 3
+	
+	static let minimumWidth: CGFloat = 120
+	
+	static let verticalPadding: CGFloat = 2
+	
+	static let horizontalPadding:  CGFloat = 2
+	
+	private init() {}
+}
+
+@objc public enum ButtonStyle: Int, CaseIterable {
+    case primaryFilled
+    case primaryOutline
+    case borderless
+}
```

Comment:
Noticed this is not publicly documented in FluentUI-iOS either, which has 5 styles
```
public enum ButtonStyle: Int, CaseIterable {
    case primaryFilled
    case primaryOutline
    case secondaryOutline
    case tertiaryOutline
    case borderless
```

Followup change may be to add those in as well, since I'm not actually sure when to use `secondaryOutline` and `tertiaryOutline`. @harrieshin Thoughts?